src/provider/model/APIExtension.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * This file defines the hidden extensions to the api that the provider uses
+ * to communicate with the notifications center, and vice-versa
+ *
+ * They are included here instead of in client/internal.ts to keep them out
+ * of any published client code.
+ */
+
+import {API} from '../../client/internal';
+
+import {StoredNotification} from './StoredNotification';
+
+export enum APITopicExtension {
```

Comment:
Something more along the lines of "notification center API", rather than an arbitrary "extension" of the client API?

Reply:
As discussed, the ideal name for this conflicts with some legacy stuff within the notification centre - will revisit when we clean-up the centre.